sphinx/tutorial/rose/furthertopics/rose-bunch.rst

```diff
@@ -131,7 +132,8 @@ directory of the lander app and running::
    chmod +x land
 
 Navigate to the top directory of your suite (where the ``suite.rc`` and
-``rose-suite.conf`` files can be found) and run ``rose suite-run``.
+:rose:file:`rose-suite.conf` files can be found) and run
+:ref`command-rose-suite-run`.
```

Comment:
Broken `:ref:` - missing colon.

Reply:
Well spotted - unfortunately this is something else sphinx doesn't raise warnings for.